### PR TITLE
Update create account screen to be GDPR compliant

### PR DESCRIPTION
**Pull request by Astach**

### CODE REVIEW

File: ui/src/main/java/com/schibsted/account/ui/login/screen/password/PasswordFragment.kt [hunk 1]
```diff
@@ -65,12 +66,24 @@ class PasswordFragment : FlowFragment<PasswordContract.Presenter>(), PasswordCon
...
             navigationListener?.onWebViewNavigationRequested(
                     WebFragment.newInstance(Routes.forgotPasswordUrl(redirectUri).toString(), uiConf?.redirectUri), LoginScreen.WEB_FORGOT_PASSWORD_SCREEN)
         }
 
+        remember_me_info.setOnClickListener {
+            navigationListener?.onDialogNavigationRequested(
+                    InformationDialogFragment.newInstance(
+                            getString(R.string.schacc_dialog_remember_me_title),
+                            getString(R.string.schacc_dialog_remember_me_description),
+                            //todo replace with correct icon
```
Comment (Nexcius) [inline ui/src/main/java/com/schibsted/account/ui/login/screen/password/PasswordFragment.kt]: Please remove this and create an issue for it

File: ui/src/main/java/com/schibsted/account/ui/login/screen/verification/VerificationFragment.java [hunk 2]
```diff
@@ -90,16 +90,34 @@ public View onCreateView(@NonNull LayoutInflater inflater, @Nullable ViewGroup c
...
+        final TextView rememberMeInfoView = view.findViewById(R.id.remember_me_info);
+        rememberMeInfoView.setOnClickListener(new View.OnClickListener() {
+            @Override
+            public void onClick(View v) {
+                if (navigationListener != null) {
+                    navigationListener.onDialogNavigationRequested(
+                            InformationDialogFragment.newInstance(
+                                    getString(R.string.schacc_dialog_remember_me_title),
+                                    getString(R.string.schacc_dialog_remember_me_description),
+                                    //todo replace with correct icon
```
Comment (Nexcius) [inline ui/src/main/java/com/schibsted/account/ui/login/screen/verification/VerificationFragment.java]: Same as above

Comment (Astach) [reply ui/src/main/java/com/schibsted/account/ui/login/screen/verification/VerificationFragment.java]: Issue is https://github.com/schibsted/account-sdk-android/issues/128

File: ui/src/main/java/com/schibsted/account/ui/login/screen/password/PasswordFragment.kt (hunk 1 above)
Comment (Astach) [reply ui/src/main/java/com/schibsted/account/ui/login/screen/password/PasswordFragment.kt]: Issue is https://github.com/schibsted/account-sdk-android/issues/128

**Pull request merged**